client/document/index.jsx

```diff
@@ -134,11 +134,7 @@ class Document extends Component {
 		}
 
 		return (
-			<html
-				lang={ lang }
-				dir={ isRTL ? 'rtl' : 'ltr' }
-				className={ clsx( { 'is-iframe': sectionName === 'gutenberg-editor' } ) }
```

Comment:
This is half of the cleanup: we can no longer be in this section since we're no longer using the iframed editor.